ui.apps/src/main/content/jcr_root/apps/acs-commons/components/workflow/copy-properties/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,114 @@
+                                            disabled="false"
+                                            fieldLabel="Description"
+                                            name="./jcr:description"/>
+                                </items>
+                            </basic>
+                            <advanced
+                                    jcr:primaryType="nt:unstructured"
+                                    jcr:title="Advanced Settings"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/fieldset">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <advance
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                            fieldDescription="Check if your handler will advance to the next step."
+                                            fieldLabel="Handler Advance"
+                                            name="./metaData/PROCESS_AUTO_ADVANCE"
+                                            text="Handler Advance"
+                                            value="true"/>
+                                </items>
+                            </advanced>
+                        </items>
+                    </column>
+                </items>
+            </processcommon>
+            <processargs
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Arguments"
+                    sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns">
+                <items jcr:primaryType="nt:unstructured">
+                    <column
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured">
+                            <arguments
+                                    jcr:primaryType="nt:unstructured"
+                                    jcr:title="Properties to Copy"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/fieldset">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <skip-empty-property
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                            text="Skip empty source properties"
+                                            fieldDescription="If the source property is empty or missing, leave the destination property alone."
+                                            named="./metadata/SKIP_EMPTY_SOURCE_PROPERTY"
```

Comment:
Is this the new convention, constant-esq metadata properties?

Reply:
This allows you to the WF Dialogs to control the input of your metaData properties; vs. PROCESS_ARGS which are the "primitive" blobs... so yeh. I've mostly not don't this before cuz I'm lazy, but i'd argue this is the "right" way to do it, and planning on using this pattern going forward.

One thing worth noting here, is the `.execute(..., MetadadaMap map)` map variable, is not your typical valuemap that can look down the tree; it can ONLY see values directly on the metadata node, which is why this property mapping isn't modeled as sub-nodes, with individual fields for source and dest.